src/reducers/SignUp.ts

```diff
@@ -46,9 +46,6 @@ export const signUp = (state: any, action: any) => {
         },
         loading: false,
       }
-    default:
-      return {
-        defaultState,
-      }
   }
+  return state || defaultState
```

Comment:
What if we add defaultState as the default value for state in line 11?
This can simply be `return state` which is more of a best practice.